app/pages/QuestionPage.scala

```diff
@@ -16,6 +16,31 @@
 
 package pages
 
+import models.UserAnswers
+import pages.addFinancialInstitution.IsRegisteredBusiness._
+import pages.addFinancialInstitution._
 import queries.{Gettable, Settable}
 
-trait QuestionPage[A] extends Page with Gettable[A] with Settable[A]
+import scala.util.Try
+
+trait QuestionPage[A] extends Page with Gettable[A] with Settable[A] {
+
+  def removePages(pages: Seq[Settable[_]], userAnswers: UserAnswers): Try[UserAnswers] =
+    pages.foldLeft(Try(userAnswers))(
+      (ua, page) => ua.flatMap(_.remove(page))
+    )
+
+  protected val allAddressPages: Seq[QuestionPage[_]] = Seq(
```

Comment:
Not sure if this is used at all anywhere

Reply:
ah it's not, thought id removed that again 👍

Reply:
removed